apps/reports/README.md

```diff
@@ -0,0 +1,24 @@
+# tables
+
+## Project setup
+```
+npm install
```

Comment:
yarn

Reply:
this is error in all apps, now only fixed here